react/components/TabbedShelf/Tabs.js

```diff
@@ -12,22 +15,22 @@ class Tabs extends Component {
   handleClick = (index) => this.setState({ selectedIndex: index })
 
   render() {
-    const { panes } = this.props
+    const { panes, cssHandles } = this.props
     const { selectedIndex } = this.state
     if (!panes || panes.length === 0) {
       return null
     }
 
     return (
       <div className="flex justify-center items-center">
-      <div className={`${tabbedShelf.tabsContainer} flex-ns pa6-ns justify-between-ns w-100-s`}>
-        <div className={`${tabbedShelf.tabsNamesContainer} flex flex-column mh6`}>
+      <div className={`${cssHandles.tabsContainer} flex-ns pa6-ns justify-between-ns w-100-s`}>
+        <div className={`${cssHandles.tabsNamesContainer} flex flex-column mh6`}>
           {panes.map((pane, index) => {
             const isSelected = index === selectedIndex
             const isLast = index === panes.length - 1
-            const itemContainer = classNames(`t-body pa4 tl bb b--muted-4 ${tabbedShelf.itemContainer}`, {
-              [`bg-base--inverted c-on-muted-1 ${tabbedShelf.itemContainerSelected}`]: isSelected,
-              [tabbedShelf.itemContainerUnselected]: !isSelected,
+            const itemContainer = classNames(`t-body pa4 tl bb b--muted-4 ${cssHandles.itemContainer}`, {
+              [`bg-base--inverted c-on-muted-1 ${cssHandles.itemContainerSelected}`]: isSelected,
+              [cssHandles.itemContainerUnselected]: !isSelected,
               'bw0': isLast,
             })
             return (
```

Comment:
You might add a new CSS handle to the button right below this line (I could not select it because it was not changed)

Reply:
I'm aware that it might not be that useful, but just to prevent anyone from asking it in the future hahahha

Reply:
Done!